notebooks/random_forest_mnmg_demo.ipynb

```diff
@@ -110,6 +110,7 @@
     "X, y = datasets.make_classification(n_samples=n_samples, n_features=n_features,\n",
```

Comment:
Can you pass float32 as the output type to make_classification instead so we don't need to make a copy with astype?

Reply:
No, because `make_classification` does not accept data type as an argument. It always returns `float64` type.

Reply:
ack, this is sklearn.datasets.make_classification...  I think we should replace that with the cuML version if possible. But that is a separate issue I'll file. This looks good given the current structure!